_pages/make/wireframing.md

```diff
@@ -6,11 +6,11 @@ redirect_from: /wireframing/
 
 ## What it is
 
-A lightweight schematic visual representation of a product or service interface.
+A simple visual representation of a product or service interface.
 
 ## Reasons to use it
 
-To prioritize information (substance and relationships) over decoration (style) as you begin defining the solution by keeping the design light. Wireframing also gives designers a great opportunity to start asking developers early questions about feasibility and structure.
+To prioritize substance and relationships over decoration as you begin defining the solution. Wireframing also gives designers a great opportunity to start asking developers early questions about feasibility and structure.
```

Comment:
I would guess information architecture, but I am not the original author. We could also simplify by deleting it.